paf-mvp-core-js/src/express/node.ts

```diff
@@ -151,21 +166,29 @@ export class Node implements INode {
    * Should be called last.
    * @param req
    * @param res
-   * @param next
+   * @param _next
    */
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  endHandling = (req: Request & { correlationId(): string }, res: Response, next: NextFunction) => {
+  endHandling = (req: Request & { correlationId(): string }, res: Response, _next: NextFunction) => {
     const { endPointName } = this.getRequestConfig(req);
     //this.logger.Info(req.rawHeaders);
     // we can get correlation-id from the request header as it was already set
     this.logger.Info(`${endPointName} --correlation-id=${req.correlationId()} - END`);
+    //End span
+    const currentSpans = res.locals.spans;
+    const currentSpan = currentSpans.pop();
+    currentSpan.setStatus({ code: res.locals.currentSpanStatus ?? SpanStatusCode.OK });
+    currentSpan.end();
   };
 
   catchErrors =
     // eslint-disable-next-line @typescript-eslint/no-unused-vars
     (error: unknown, req: Request & { correlationId(): string }, res: Response, next: NextFunction) => {
       const { endPointName, isRedirect } = this.getRequestConfig(req);
 
+      this.logger.Error(endPointName, error);
```

Comment:
Any reason to add this log line? There's already one with specific formatting line 241

Reply:
Removed.